web/module/uptime.py

```diff
@@ -1,15 +1,17 @@
 import datetime
-from sanic import Blueprint
 import sanic.response
+from sanic import Blueprint
 import socket
 
 uptime = Blueprint('uptime')
 
 
 @uptime.route('/uptime')
-async def uptime(request):
+async def uptime_f(request):
```

Comment:
What is the meaning of _f in the name? Is it for "fast"?

Reply:
because it cannot be uptime so I added some suffix